### DISCUSSION TITLE
Which tools you use for visualisation and statistics on you pc?

### DISCUSSION THREAD

**Discussion by c33s** [OpenTracksApp/OpenTracks#1667] [Q&A]
in the discussion https://github.com/OpenTracksApp/OpenTracks/discussions/1555 i got informed that you should use opentracks only for tracking and another tool for visalisation of the collected data.

i would be glad to hear any tips for good software to do so (opensource would be great)

**Comment by dennisguse** [comment]
Some people use a 
* NextCloud plugin
* https://github.com/SamR1/FitTrackee
* Gnome Maps :D

That's all I am aware off.

I always wanted to document this, but didn't take the time to do this: #1330

**Comment by c33s** [comment]
GoldenCheetah https://github.com/GoldenCheetah/GoldenCheetah

**Comment by MartinH80** [comment]
I tried [Sportstracker](https://github.com/ssaring/sportstracker/releases), but currently have an [issue with the import of the gpx file](https://github.com/ssaring/sportstracker/issues/260).

**Reply by dennisguse** [reply]
Does KML/KMZ work?
@MartinH80 can you extend our README.md with your findings?

**Reply by MartinH80** [reply]
Yes, will do. Let's see if sportstracker can solve the import soon so I can report that it works. I will also test with KMZ files.
